shared/models.go

```diff
@@ -40,3 +44,56 @@ type Browser struct {
 type Token struct {
 	Secret string `json:"secret"`
 }
+
+type Manifest struct {
+	Items   ManifestItems `json:"items,omitempty"`
+	Version *int          `json:"version,omitempty"`
```

Comment:
Why is this a pointer?

Reply:
JSON marshalling in Go will only drop a field if it's a pointer, otherwise it's still marshalled as the empty/default value of the struct. 

Note that unmarshalling in this fashion is unaffected.